app/src/main/java/com/framgia/forder/screen/createshop/CreateshopPresenter.java

```diff
@@ -1,13 +1,20 @@
 package com.framgia.forder.screen.createshop;
 
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
```

Comment:
ERROR: Unused import: android.net.Uri. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports